feedback/feedback.py

```diff
@@ -102,141 +117,81 @@ def resource_string(cls, path):
         data = pkg_resources.resource_string(__name__, path)
         return data.decode("utf8")
 
-    def get_prompt(self, index=-1):
-        """
-        Return the current prompt dictionary, doing appropriate
-        randomization if necessary, and falling back to defaults when
-        necessary.
-        """
-        if index == -1:
-            index = self.prompt_choice
-
-        _ = self.runtime.service(self, 'i18n').ugettext
-        # This is the default prompt if something is not specified in the
-        # settings dictionary. Note that this is not the same as the default
-        # above. The default above is the prompt the instructor starts from
-        # in a tool like Studio. This is a fallback in case some JSON fields
-        # are left unpopulated (e.g. if someone manually tweaks the database,
-        # in case of OLX authoring, and similar). The examplar above is
-        # intended as a well-structured, coherent response. This is designed
-        # as generic, to work with any content as a safe fallback.
-        prompt = {
-            'freeform': _("Please reflect on this course material"),
-            'default_text': _("Please take time to meaningfully reflect "
-                              "on your experience with this course "
-                              "material."),
-            'likert': _("Please rate your overall experience"),
-            'scale_text': [_("Excellent"),
-                           _("Good"),
-                           _("Average"),
-                           _("Fair"),
-                           _("Poor")],
-            'icon_set': 'num',
-            'placeholder': "Please take a moment to thoughtfully reflect."
-        }
-
-        prompt.update(self.prompts[index])
-        return prompt
-
     def student_view(self, context=None):
         """
         The primary view of the FeedbackXBlock, shown to students
         when viewing courses.
         """
-        # Figure out which prompt we show. We set self.prompt_choice to
-        # the index of the prompt. We set it if it is out of range (either
+        # Figure out which prompts we show. We set self.prompts_choice to
+        # the index of the prompts. We set it if it is out of range (either
         # uninitiailized, or incorrect due to changing list length). Then,
-        # we grab the prompt, prepopulated with defaults.
-        if self.prompt_choice < 0 or self.prompt_choice >= len(self.prompts):
-            self.prompt_choice = random.randint(0, len(self.prompts) - 1)
-        prompt = self.get_prompt()
+        # we grab the prompts, prepopulated with defaults.
+        propts_size = ASSESSMENTS_NUMBER
```

Comment:
Why do we use here additional variable?

Reply:
as range of assessments is used ASSESSMENTS_NUMBER variable
 I think that is easier to understand and use local variable which compatible with context of this method code